### PR TITLE
Add warning for NFS + Zeebe

### PR DESCRIPTION
**Pull request by akeller**
## Description

A quick win for #2987, but will not close out the issue entirely.

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [ ] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
@oleschoenburg @npepinpe, I added a warning to the main Zeebe installation page in Self-Managed, but I have a few clarifying questions:

* Is there a better place/page you would recommend to add this or cross link? 
* We have a [known limitations topic](#1042 ) in the docs backlog, but it looks like both topics are more general for both SaaS and SM. Would adding the known limitations page now help you revisit this topic in more detail in the future?
* Would this be backported to all supported versions? (Seems like yes)

File: docs/self-managed/zeebe-deployment/zeebe-installation.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ title: "Overview"
 sidebar_label: "Overview"
 ---
 
+:::warning
+Zeebe does not support network files systems (NFS) at this time.
```
Comment (oleschoenburg) [inline docs/self-managed/zeebe-deployment/zeebe-installation.md:8]:
```suggestion
Zeebe does not support network files systems (NFS) at this time. Usage of NFS may cause data corruption.
```

**Review by oleschoenburg — DISMISSED** [review]
> Is there a better place/page you would recommend to add this or cross link?

I can't think of another place that would be a better fit.

> Would adding the known limitations page now help you revisit this topic in more detail in the future?

This would certainly be a good place to document this :+1: 

> Would this be backported to all supported versions? (Seems like yes)

Yes, this warning applies to all versions of Zeebe

File: docs/self-managed/zeebe-deployment/zeebe-installation.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ title: "Overview"
 sidebar_label: "Overview"
 ---
 
+:::warning
+Zeebe does not support network files systems (NFS) at this time. Usage of NFS may cause data corruption.
```
Comment (akeller) [inline docs/self-managed/zeebe-deployment/zeebe-installation.md:8]:
```suggestion
Zeebe does not support network file systems (NFS) at this time. Usage of NFS may cause data corruption.
```

**Review by christinaausley — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**